diff --git a/gulpfile.js b/gulpfile.js
index ffb38e0..7b170ac 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -22,8 +22,7 @@ var gulp = require('gulp'),
     googlecdn = require('gulp-google-cdn'),
     replace = require('gulp-replace'),
     less = require('gulp-less'),
-    ftp = require('vinyl-ftp'),
-    gzip = require('gulp-gzip');
+    ftp = require('vinyl-ftp');
 
 
 var path = {
@@ -108,7 +107,7 @@ gulp.task('less',function(){
             gulp.src(mainBowerFiles('**/*.less'),{ base: './src/bower_components' })
         .pipe(less())
         .pipe(gulp.dest(path.build.css)) //И в build
-})
+});
 gulp.task('js:build', function () {
     gulp.src(path.src.js) //Найдем наш main.js файл
         .pipe(rigger()) //Прогоним через rigger
diff --git a/src/index.html b/src/index.html
index bf57996..7ce52b6 100644
--- a/src/index.html
+++ b/src/index.html
@@ -10,7 +10,7 @@
 		<!-- endbower -->
 		<link rel="stylesheet" href="css/main.css" />
 	</head>
-	<body class="hold-transition skin-blue sidebar-mini">
+	<body class="hold-transition layout-boxed skin-blue sidebar-mini">
 		<div class="wrapper">
 
 			//= template/header.html
@@ -19,7 +19,6 @@
 
 			//= template/page_content.html
 		
-			
 			//= template/footer.html
 
 			//= template/control_sidebar.html
diff --git a/src/js/main.js b/src/js/main.js
index e95e53c..fd9d141 100644
--- a/src/js/main.js
+++ b/src/js/main.js
@@ -6,4 +6,5 @@
  * Custom
  */
 //= partials/time_dyn.js
-//= partials/status.js
\ No newline at end of file
+//= partials/status.js
+//= partials/inputs.js
\ No newline at end of file
diff --git a/src/js/partials/status.js b/src/js/partials/status.js
index 1998a34..778a8be 100755
--- a/src/js/partials/status.js
+++ b/src/js/partials/status.js
@@ -11,7 +11,7 @@ function status_update()
 	for (var i = 0; i < cpunames.length; i++) 
 	{
 		var element = document.getElementsByClassName(cpunames[i]);
-		if(element)
+		if(element.length>0)
 		{
 			exist++;
 		}
diff --git a/src/js/partials/time_dyn.js b/src/js/partials/time_dyn.js
index 7e85104..6849ec3 100755
--- a/src/js/partials/time_dyn.js
+++ b/src/js/partials/time_dyn.js
@@ -8,17 +8,17 @@ function localtime_update()
 {
 	var exist=0;
 	var element=document.getElementsByClassName("date_and_time");
-	if(element)
+	if(element.length>0)
 		{
 			exist++;
 		}
 	var element=document.getElementsByClassName("time");
-	if(element)
+	if(element.length>0)
 		{
 			exist++;
 		}
 	var element=document.getElementsByClassName("date");
-	if(element)
+	if(element.length>0)
 		{
 			exist++;
 		}
diff --git a/src/template/handlebars.html b/src/template/handlebars.html
index beaccd9..2ed1838 100644
--- a/src/template/handlebars.html
+++ b/src/template/handlebars.html
@@ -63,3 +63,18 @@
 	{{memory.usage}}
 	</div>
 </script>
+
+<script id="dimmer_template" type="text/x-handlebars-template">
+<div>
+	{{#each inputs}}
+		{{#ifCond type '==' "dimmer" }}
+		<div class="col-xs-6 col-md-3 text-center">
+          <div>
+          	{{#makeDimmer}}{{/makeDimmer}}
+          </div>
+          <div class="knob-label"><h5>{{name}}</h5>{{place}}<h6>{{serial}}</h6></div>
+        </div>
+		{{/ifCond}}
+	{{/each}}
+</div>
+</script>
diff --git a/src/template/page_content.html b/src/template/page_content.html
index f1f02cc..5dff7c0 100644
--- a/src/template/page_content.html
+++ b/src/template/page_content.html
@@ -16,7 +16,9 @@
     
     <!-- Your Page Content Here -->
     <section class="content">
-      //= server_info.html
+      //= server_box.html
+      //= input_box.html
+
     </section>
     
   </section>
